### PR TITLE
Misc fixes mainly for 1893

### PR DESCRIPTION
**Pull request by perwestling**
MInor fixes for 1893.

As 1893 uses two 20% certificates (in AGV and HGK) I have generalized the 'd' shares handling of 1849. last_cert is still kept but I have added a double_cert attribute that is used in 1849 and 1893 to show 'd'.

1893 HGK:
image

1849:
image

### CODE REVIEW

File: lib/engine/g_1893/corporation.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require_relative '../corporation'
+
+module Engine
+  module G1893
+    class Corporation < Corporation
```
Comment (tobymao) [inline lib/engine/g_1893/corporation.rb:7]: delete this class

File: lib/engine/game/g_1893.rb [hunk 2]
```diff
@@ -104,6 +121,24 @@ def remove_tiles(tiles)
         end
       end
 
+      def init_corporations(_stock_market)
+        corporations = CORPORATIONS.dup
+
+        corporations.map! do |corporation|
+          Engine::G1893::Corporation.new(
```
Comment (tobymao) [inline lib/engine/game/g_1893.rb:128]: delete this and just set the properties appropriately

**Comment by tobymao** [comment]
compress the logos

File: lib/engine/g_1893/corporation.rb (hunk 1 above)
Comment (perwestling) [reply lib/engine/g_1893/corporation.rb:7]: Ok. I have moved the floatable attribute into the general corporation class.

File: lib/engine/game/g_1893.rb (hunk 2 above)
Comment (perwestling) [reply lib/engine/game/g_1893.rb:128]: Moved most of what was done in constructor to the setup class of 1893.
Have removed this part.

**Comment by perwestling** [comment]
@tobymao New version of this PR.

**Review by tobymao — APPROVED** [review]

**Pull request merged**